app/controllers/api/v0/admins_controller.rb

```diff
@@ -8,6 +8,15 @@ def add_portfolio
         render :json => { :errors => e.message }, :status => :unprocessable_entity
       end
 
+      def edit_portfolio
+        portfolio = Portfolio.find(params.require(:portfolio_id))
+        portfolio.update!(portfolio_params)
+
+        render :json => portfolio
+      rescue ActiveRecord::RecordInvalid => e
```

Comment:
@syncrou I don't think we have to handle this, rails already manages this
from a rails console if you ran
ActionDispatch::ExceptionWrapper.rescue_responses
You would see "ActiveRecord::RecordInvalid"=>:unprocessable_entity in the list

Reply:
@mkanoor - I agree, I added it because of the same code on line 7 above it: https://github.com/ManageIQ/service_portal-api/blob/79370608f32e1c2da21f4504d2c33445ffc7085f/app/controllers/api/v0/admins_controller.rb#L7-L9

My point in adding it was to make sure the caller received a json response with the specific error.